_config.yml

```
@@ -1,6 +1,7 @@
 title: FarmLife-Wiki
 description: "Wiki für FarmLife: Badoras!"
 theme: just-the-docs
+favicon_ico: "/assets/images/favicon.ico"
```

```suggestion
favicon_ico: "/assets/images/favicon.png"
```

Btw witzig: Das Bild war schon im Repo, hab ich übersehen.

---

Habs eigtl grad hochgeladen :D

---

Allerdings das falsche. Wollte das .ico nehmen. Egal, sollte auch so funzen

---

Achso. Ja, dann lad doch lieber das .ico hoch, wenn das existiert.

---

done

---

Ah, du hast das auf main direkt gemacht. Weiß Bescheid.